cluster/addons/fluentd-elasticsearch/fluentd-es-image/Dockerfile

```diff
@@ -19,8 +19,10 @@
 # Note that fluentd is run with root permssion to allow access to
 # log files with root only access under /var/log/containers/*
 
-FROM gcr.io/google-containers/debian-base-amd64:0.1
+FROM debian:stretch-slim
 
+COPY clean-apt /usr/bin
```

Comment:
You mean `clean-install`? `clean-apt` has never existed prior to this PR

It's in the previous base image: https://github.com/kubernetes/kubernetes/tree/master/build/debian-base